### ISSUE TITLE
Cleaner does not honor/copy output settings

### ISSUE THREAD

**Issue by sebhofmann** [jhy/jsoup#1417]
If you Clean a document which has output settings defined, then the result document does not have these settings set.

**Comment by sebhofmann** [comment]
Code to reproduce:
```
    public static void main(String[] args) {
        final Document test = Jsoup.parse("<p>test<br></p>");
        test.outputSettings().syntax(Document.OutputSettings.Syntax.xml);
        test.outputSettings().escapeMode(Entities.EscapeMode.xhtml);
        final Whitelist whitelist = Whitelist.none().addTags("p", "br");
        final Document result = new Cleaner(whitelist).clean(test);
        System.out.println(result.html());
    }
```

Expected Result
```
<html>
 <head></head>
 <body>
  <p>test<br /></p>
 </body>
</html>
```

Actual Result
```
<html>
 <head></head>
 <body>
  <p>test<br></p>
 </body>
</html>
```

**Comment by jhy** [comment]
Thanks, this was implemented with 532b79a61ae8191e2927c59300af68d18a54aa4e
